src/contour/display/TerminalDisplay.h

```diff
@@ -44,6 +44,7 @@ class TerminalDisplay: public QQuickItem
     Q_PROPERTY(TerminalSession* session READ getSessionHelper WRITE setSession NOTIFY sessionChanged)
     Q_PROPERTY(QString profile READ profileName WRITE setProfileName NOTIFY profileNameChanged)
     Q_PROPERTY(QString title READ title NOTIFY titleChanged)
+    Q_PROPERTY(QQuickWindow* window READ window)
```

Comment:
I think this is not used. Why are we adding it? Is it a leftover of some test code?